### PR TITLE
Base editor screen

### PR DESCRIPTION
**Pull request by Shawdooow**
Just adds the most basic editor screen, not really sure what it should have since I figured that each component in it should be its own class?

### CODE REVIEW

File: osu.Game/Screens/Edit/EditorScreen.cs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        public EditorScreen()
+        {
+            Children = new Drawable[]
+            {
+                new TwoLayerButton
+                {
+                    Anchor = Anchor.BottomRight,
+                    Origin = Anchor.BottomRight,
+                    Colour = Color4.Yellow,
+                    Action = delegate {Push(songSelect);},
```
Comment (VPeruS) [inline osu.Game/Screens/Edit/EditorScreen.cs]: spaces

File: osu.Game/Screens/Edit/EditorScreen.cs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                changeBackground(currentBeatmap);
+            }
+            songSelect = new EditSongSelect();
+            Beatmap.Value.Track?.Stop();
+            base.OnResuming(last);
+        }
+        protected override void OnEntering(Screen last)
+        {
+            base.OnEntering(last);
+            if (songSelect ?.SelectedBeatmap != null)
```
Comment (jorolf) [inline osu.Game/Screens/Edit/EditorScreen.cs]: why is there a space? and isn't songSelect always null when entering this screen?

File: osu.Game/Screens/Menu/MainMenu.cs [hunk 3]
```diff
@@ -47,7 +47,7 @@ public MainMenu()
                         {
                             OnChart = delegate { Push(new ChartListing()); },
                             OnDirect = delegate { Push(new OnlineListing()); },
-                            OnEdit = delegate { Push(new Editor()); },
+                            OnEdit = delegate { Push(new EditorScreen()); },
```
Comment (peppy) [inline osu.Game/Screens/Menu/MainMenu.cs]: why rename? nothing else is using a `Screen` suffix.

**Comment by peppy** [comment]
I mean this PR isn't really doing much at all. I'd suggest starting to implement actual components rather than an empty screen

**Comment by smoogipooo** [comment]
Closing for now - see previous comments.

**Pull request closed without merging**